compiler/dyno/lib/parsing/ParserContextImpl.h

```diff
@@ -785,6 +786,173 @@ FunctionParts ParserContext::makeFunctionParts(bool isInline,
   return fp;
 }
 
+static AstNode*
+findErrorInFnBodyOrFormals(ParserContext* context, YYLTYPE location,
+                           FunctionParts& fp) {
+  if (fp.errorExpr) return fp.errorExpr;
+  if (fp.formals) {
+    for (auto ast : *fp.formals) {
+      if (!ast->isErroneousExpression()) continue;
+      auto loc = context->convertLocation(location);
+      auto errorExpr = ErroneousExpression::build(context->builder, loc);
+      fp.errorExpr = errorExpr.release();
+      return fp.errorExpr;
+    }
+  }
+
+  return nullptr;
+}
+
+AstNode*
+ParserContext::buildFunctionExpr(YYLTYPE location, FunctionParts& fp) {
+  if (fp.isBodyNonBlockExpression) assert(false && "Not handled!");
```

Comment:
Do we want to address this before merging the PR, or do you think it's fine to get this working later?

Reply:
I think it's fine to get it working later. The whole effort should be considered `--warn-unstable`.